src/main/java/dk/cngroup/lentils/entity/FinalPlace.java

```diff
@@ -0,0 +1,92 @@
+package dk.cngroup.lentils.entity;
+
+import org.springframework.data.geo.Point;
+
+import javax.persistence.*;
+import javax.validation.Constraint;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "finalPlace")
+public class FinalPlace {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @Size(min = 5, max = 50)
+    @Column(name = "placeTitle")
+    private String placeTitle;
```

Comment:
Prejmenoval bych to na "title", jinak to vede ke konstrukci finalPlace.getPlaceTitle() pricemz uz class/nazev promenne uz obsahuje to "place".

Reply:
Klidně bych  nechal jen title. Když je to v té třídě FinalPlace tak je celkem jasné, k čemu se to vztahuje.